nixos/modules/services/mail/postsrsd.nix

```diff
@@ -7,6 +7,19 @@
 let
 
   cfg = config.services.postsrsd;
+  runtimeDirectoryName = "postsrsd";
+  runtimeDirectory = "/run/postsrsd";
+  # <<< TODO: follow RFC 42, need a libconfuse format first >>>
```

Comment:
I'm not sure if we already have `pkg.formats` support for `libconfuse`. I've asked about that here: https://discourse.nixos.org/t/is-there-a-way-to-use-pkgs-formats-with-libconfuse/62836.

I'm happy to try to add libconfuse support for `pkgs.formats`, but I'd like to get feedback on this PR before spending time doing that.

Reply:
No support yet, see `pkgs/pkgs-lib/formats.nix`.

Reply:
Thanks. My preference would be to get (some version of) this PR merged up, and then follow up later with proper RFC 42 support (which is blocked by adding libconfuse support to `pkgs.format`).

@abbradar, how would you like me to proceed here?

Reply:
I'm going to mark this resolved.

I have done some research on what it would take to add proper `pgks.formats` support for libconfuse, and it's... nontrivial. I've filed https://github.com/NixOS/nixpkgs/issues/401565 with details about what I've learned. I'll keep pushing on it in the background, but I'm now convinced we should merge this PR as is, and follow up with more flexible configuration in the future.